src/OpenLoco/Scenario.cpp

```diff
@@ -142,9 +150,25 @@ namespace OpenLoco::Scenario
     // 0x0049685C
     void initialiseDate(uint16_t year)
     {
-        registers regs;
-        regs.ax = year;
-        call(0x0049685C, regs);
+        // NB: this base value was already 1800 in Locomotion.
+        uint32_t ebx = year - 1800;
+        uint32_t eax = (ebx >> 2) * 1462;
```

Comment:
Not sure what 1462 represents. Thoughts?

Reply:
1461 is 365*3 + 366 you sure it's 1462

Reply:
It is indeed 1461 at 0x0049686C. Nice catch.